packages/api/src/routes/oauth/callback.post.ts

```diff
@@ -2,23 +2,20 @@ import { OAuth2Routes, type RESTPostOAuth2AccessTokenResult, type RESTPostOAuth2
 import { stringify } from 'querystring';
 import { fetch } from 'undici';
 import { Route } from '../../lib/structures/Route';
-import type { ApiRequest } from '../../lib/structures/api/ApiRequest';
-import type { ApiResponse } from '../../lib/structures/api/ApiResponse';
 import { HttpCodes } from '../../lib/structures/http/HttpCodes';
-import { methods } from '../../lib/structures/http/HttpMethods';
 
 export class PluginRoute extends Route {
 	private readonly redirectUri: string | undefined;
 
 	public constructor(context: Route.LoaderContext) {
-		super(context, { route: 'oauth/callback' });
+		super(context, { route: 'oauth/callback', methods: ['POST'] });
```

Comment:
Why do we need methods here? I thought it was inferred from the file name

Reply:
Core and plugin packages use virtual pieces, so those need to be specified explicitly.